newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/index.js

```diff
@@ -427,25 +427,27 @@ const BuildSection = ({
             </Column>
           </Line>
         ) : (
-          <ListItem style={styles.listItem}>
-            <Column expand>
-              <Paper
-                variant="outlined"
-                background="dark"
-                style={styles.noProjectsContainer}
-              >
-                <BackgroundText>
-                  <Trans>No projects yet.</Trans>
-                </BackgroundText>
-                <BackgroundText>
-                  <Trans>
-                    Create your first project using one of our templates or
-                    start from scratch.
-                  </Trans>
-                </BackgroundText>
-              </Paper>
-            </Column>
-          </ListItem>
+          <Line>
```

Comment:
Why this change? A display problem? Do the added margins look ok?

Reply:
Yes, on mobile & desktop the empty message is too close to the buttons:
![image](https://github.com/4ian/GDevelop/assets/4895034/065cb3d3-5a90-49ea-81e9-cff4aab32215)